### DISCUSSION TITLE
fish for VSCode Integrated Terminal

### DISCUSSION THREAD

**Discussion by crstxkng** [end-4/dots-hyprland#400] [Q&A] [answered]
Hi! I'm a little newbie
I want to fix the problem of my Visual Studio Code integrated terminal font

Fish is using Noto Sans Regular but in VSCode I'm in trouble with the 'icons' for the current directory and git branch. I mean this is what I expect: 
![image](https://github.com/end-4/dots-hyprland/assets/82126489/ddb6e471-64f3-4a5c-a6ad-0afab7bea742)

But I get this in VSCode:
![image](https://github.com/end-4/dots-hyprland/assets/82126489/2efe772d-d501-4e69-aed0-fe1ce069b58c)

I consider this warning:
![image](https://github.com/end-4/dots-hyprland/assets/82126489/5fd03b94-de79-4d6f-b373-b400ff9b73ca)

Any ideas to solve this?
Thanks in advance

**Comment by end-4** [comment, marked as answer]
go to your vscode settings, search for font family and change it to a nerd font

**Reply by crstxkng** [reply]
Thanks, I get it with `JetBrainsMonoNL Nerd Font`  

![image](https://github.com/end-4/dots-hyprland/assets/82126489/85576891-16d3-472d-af56-42bf58f44a27)
